tests/integration-tests/tests/common/utils.py

```diff
@@ -105,9 +108,8 @@ def retrieve_pcluster_ami_without_standard_naming(region, os, version, architect
             Filters=[
                 {"Name": "name", "Values": [official_ami_name]},
                 {"Name": "architecture", "Values": [architecture]},
-                {"Name": "is-public", "Values": ["true"]},
             ],
-            Owners=OS_TO_PCLUSTER_AMI_NAME_OWNER_MAP.get(os).get("owners"),
+            Owners=["self"],
```

Comment:
Should we also include the official ones?

Reply:
No. We need to call `copy_image`, which leads to permission error if the source AMI is official